knotx-example/pom.xml

```diff
@@ -0,0 +1,122 @@
+<!--
+
+    Knot.x - Sample App with Mock service
+
+    Copyright (C) 2016 Cognifide Limited
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.cognifide.knotx</groupId>
+        <artifactId>knotx-root</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>knotx-example</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <name>Knot.x - Sample App with Mock service</name>
+
+    <inceptionYear>2016</inceptionYear>
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Tomasz Michalak</name>
+            <email>tomasz.michalak@cognifide.com</email>
+        </developer>
+        <developer>
+            <name>Maciej Laskowski</name>
+            <email>maciej.laskowski@cognifide.com</email>
+        </developer>
+    </developers>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-configuration-processor</artifactId>
+            <optional>true</optional>
+        </dependency>
+
+        <!-- Knot.x -->
+        <dependency>
+            <groupId>com.cognifide.knotx</groupId>
+            <artifactId>knotx-core</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
We could use `${poject.version}` here (in order to not have to update this later).